packages/create-rslib/fragments/base/vue-js/tsconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "lib": ["DOM", "ES2021"],
+    "module": "ESNext",
+    "jsx": "react-jsx",
```

Comment:
Should be
```ts
    "jsx": "preserve",
    "jsxImportSource": "vue",
```
The same goes for the other tsconfigs

Reply:
We can copy from https://github.com/web-infra-dev/rsbuild/blob/main/packages/create-rsbuild/template-vue3-ts/tsconfig.json